src/Events/Calendar_Embeds/Admin/Singular_Page.php

```diff
@@ -85,12 +85,17 @@ public function unregister(): void {
 	 * Modifies the post updated messages for the calendar embed post type.
 	 *
 	 * @since 6.11.0
+	 * @since TBD Made the parameters non-strict.
 	 *
 	 * @param array $messages The post updated messages.
 	 *
 	 * @return array
 	 */
-	public function modify_post_updated_messages( array $messages ): array {
+	public function modify_post_updated_messages( $messages ): array {
+		if ( ! is_array( $messages ) ) {
```

Comment:
I didn't look at where the filter for this one is, but is there a special case when some other hook has modified `$messages` and therefore we want to skip our logic entirely? Wouldn't we want to instead do something like `$messages = (array) $messages;`, and then continue our logic?

Reply:
Yea that is fine. what was happening they would set $messages to null. Giving me the idea that they dont want messages to be set at all.

They can fix their logic to be compatible with what WP expects. I decided to simply bail in this case which is ok